best_practices/api_call_sequencing.md

```diff
@@ -0,0 +1,51 @@
+---
+permalink: /api-call-sequencing
+title: App Installation Design and API Call Sequencing
```

Comment:
@jimj-procoretech I've never heard the tearm "API Call Sequencing". What does it mean?

Reply:
Quite a familiar term to me actually, though my development background might be considered 'old' and rooted in the early 2000's world of object models, sequence diagrams, etc. For example...  https://www.ibm.com/docs/en/ftmfz/3.0.5?topic=api-call-sequence
But if there is a more 'modern' way of conveying this I'd like to learn about it.

Reply:
As far as I know, we don't have this limitation in Procore API. You can call any endpoint at any time.
Is that not the case?

Reply:
That is indeed the case. It is the order in which they are called that can have undesired effects on app installation. I've updated the main description of the PR with more detail and provided links to supporting materials.

Reply:
Let me see if I understood correctly.

What this page is saying "Dear developer, when a Procore user authorizes your app, the admins of the companies the user belong to will not have any visibility over that. They will only have visibility after you make any company-specific API request on behalf of that user."

Is this the core message of this page?